internal/pkg/cli/selector/selector.go

```diff
@@ -140,6 +140,28 @@ func (s *Select) Environment(prompt, help, app string) (string, error) {
 	return selectedEnvName, nil
 }
 
+// EnvironmentWithNone fetches all the environments in an app and prompts the user to select one of the environments or None.
+func (s *Select) EnvironmentWithNone(prompt, help, app string) (string, error) {
+	envs, err := s.retrieveEnvironments(app)
+	if err != nil {
+		return "", fmt.Errorf("get environments for app %s from metadata store: %w", app, err)
```

Comment:
Can we add a unit test that will hit this error block?

Reply:
Good call..! I realized that `EnvironmentWithNone` and `Environment` wrap the error returned from `retrieveEnvironments`, which is from the same package. Is this wrapping necessary? I feel like I can just return "", err for line 147.

Reply:
I think this error can be passed through like you say instead of being wrapped.